db_management/db_online.rst

```diff
@@ -7,45 +7,75 @@
 Online Database management
 ==========================
 
+To manage your databases, access the `database management page <https://www.odoo.com/my/databases>`__
+(you will have to sign in). Then click on the
+`Manage Your Databases <https://www.odoo.com/my/databases/manage>`__ button.
+
+.. image:: media/databases.png
+    :align: center
+
+Make sure you are connected as the Administrator of the database you
+want to manage - many operations depends on indentifying you remotely to that database.
```

Comment:
80 char limit per page you shall respect (and all the occurrences below)

Reply:
Well shit, I told @efaOdoo the limit was 100 -_-